summernote/summernote
# BS4 & BS3 Updates

#### What does this PR do?
Appologies for the multiple Pull-Requests I did not realize that BS4 was updated and have closed the past PR. This PR is just two minor issues with the current dev branch.


- BS3 and [BS4 ](https://getbootstrap.com/docs/4.0/components/forms/#checkboxes-and-radios-1) has different markup for checkboxes (notably the checkbox in the LinkDialog)
- Fixes the "x" (close) element on dialogs

#### Where should the reviewer start?

- Review the code in both src/bs3/ui.js & src/bs4/ui.js added checkbox independant ui rendering.
- Review the LinkDialog.js to look at the code that renders the checkbox based on the ui renderer.

#### How should this be manually tested?

- Open the link dialog in both BS3 and BS4 and should be rendered per-framework.
- Ensure the BS4 close (x) is on the right

#### Screenshots 
Bootstrap 4:
![Bootstrap 4 Modal with Checkbox](https://user-images.githubusercontent.com/11223218/29955363-48d38b6c-8f1e-11e7-8d81-a21d2af531f0.png)

Bootstrap 3:
![Bootstrap 3 Modal with Checkbox](https://user-images.githubusercontent.com/11223218/29955380-6358588c-8f1e-11e7-8c9f-401b27990c16.png)
